task/task_test.go

```diff
@@ -77,6 +77,7 @@ func TestTask(t *testing.T) {
 					Script: `#!/bin/bash
 						cat data
 						echo "$ENVIRONMENT_VARIABLE_DATA" | tee data
+						sleep 60
```

Comment:
```suggestion
						cat data
						echo "$ENVIRONMENT_VARIABLE_DATA" > data
						sleep 60
						cat data
```

Otherwise, it would be ineffective: the condition to terminate the machine is the presence of bith the old and the new data strings.